presto-main/src/test/java/com/facebook/presto/sql/planner/TestQuantifiedComparison.java

```diff
@@ -65,29 +58,25 @@ public void testQuantifiedComparisonNotEqualsAll()
     @Test
     public void testQuantifiedComparisonLessAll()
     {
-        assertOrderedQuantifiedComparison("SELECT orderkey, custkey FROM orders WHERE orderkey < ALL (VALUES CAST(5 as BIGINT), CAST(3 as BIGINT))",
-                "X < MIN", "X", "min", "MIN");
+        assertQuantifiedComparison("SELECT orderkey, custkey FROM orders WHERE orderkey < ALL (VALUES CAST(5 as BIGINT), CAST(3 as BIGINT))");
     }
 
     @Test
     public void testQuantifiedComparisonGreaterEqualAll()
     {
-        assertOrderedQuantifiedComparison("SELECT orderkey, custkey FROM orders WHERE orderkey >= ALL (VALUES CAST(5 as BIGINT), CAST(3 as BIGINT))",
-                "X >= MAX", "X", "max", "MAX");
+        assertQuantifiedComparison("SELECT orderkey, custkey FROM orders WHERE orderkey >= ALL (VALUES CAST(5 as BIGINT), CAST(3 as BIGINT))");
     }
 
     @Test
     public void testQuantifiedComparisonLessSome()
     {
-        assertOrderedQuantifiedComparison("SELECT orderkey, custkey FROM orders WHERE orderkey < SOME (VALUES CAST(5 as BIGINT), CAST(3 as BIGINT))",
-                "X < MAX", "X", "max", "MAX");
+        assertQuantifiedComparison("SELECT orderkey, custkey FROM orders WHERE orderkey < SOME (VALUES CAST(5 as BIGINT), CAST(3 as BIGINT))");
     }
 
     @Test
     public void testQuantifiedComparisonGreaterEqualAny()
     {
-        assertOrderedQuantifiedComparison("SELECT orderkey, custkey FROM orders WHERE orderkey >= ANY (VALUES CAST(5 as BIGINT), CAST(3 as BIGINT))",
-                "X >= MIN", "X", "min", "MIN");
+        assertQuantifiedComparison("SELECT orderkey, custkey FROM orders WHERE orderkey >= ANY (VALUES CAST(5 as BIGINT), CAST(3 as BIGINT))");
```

Comment:
Those were asserting quite a lot about implementation while not giving more confidence correctness-wise than the result tests. Also, I think you realize the plan matcher dsl isn't flexible enough for complex expressions (like case expressions used in that particular plan)